froggah.py

```diff
@@ -68,7 +68,7 @@
     cars.append(randint(1, 9))
 distance = []
 for r in range(rows):
-    distance.append(screenx / cars [r])
+    distance.append((screenx + carwidth) // cars [r])

```

Comment:
necessary to adapt to the "larger screen" for the cars in order to make their apparition smoother